src/scene/particle-system/particle-emitter.js

```diff
@@ -392,7 +392,10 @@ Object.assign(pc, function () {
 
             this.worldBoundsSize.copy(this.worldBounds.halfExtents).scale(2);
 
-            this.meshInstance.mesh.aabb = this.worldBounds;
+            this.meshInstance.aabb.setFromTransformedAabb(
+                this.worldBounds, this.localSpace ? this.node.getWorldTransform() : pc.Mat4.IDENTITY);
+            this.meshInstance.mesh.aabb.setFromTransformedAabb(
+                this.worldBounds, this.localSpace ? this.node.getWorldTransform() : pc.Mat4.IDENTITY);
```

Comment:
If it's not local space, can't worldBounds just be copied the the meshInstance's aabb? Why call setFromTransformedAabb at all if the transform is identity?